README.md

```diff
@@ -96,13 +96,16 @@ available [below](#output-format).
 * Response body content
     * Saves all files encountered during the crawl to a `LevelDB`
         database de-duplicated by the md5 hash of the content.
-    * Set `browser_params['save_all_content'] = True`
+    * Set `browser_params['save_content'] = True`
     * The `content_hash` column of the `http_responses` table contains the md5
         hash for each script, and can be used to do content lookups in the
         LevelDB content database.
     * NOTE: this instrumentation may lead to performance issues when a large
         number of browsers are in use.
-    * Set `browser_params['save_javascript'] = True` to save only Javascript
+    * Set `browser_params['save_content']` to a comma-separated list of 
```

Comment:
In your PR summary you mention that you can also initialize this with a list of strings. Should we document that here?

Reply:
If this is an acceptable config type, be sure to check for it in the misconfiguration checks described in my comment above. I think it's better to only support a single multi-option config format (either comma-separated string or a list of strings).

Reply:
> I think it's better to only support a single multi-option config format (either comma-separated string or a list of strings).

Agree. That is simpler. Removed support for list of strings.